### DISCUSSION TITLE
I want to install the CARLA but I am facing the issues many time.

### DISCUSSION THREAD

**Discussion by Aimo2027** [carla-simulator/carla#8363] [Q&A]
I'm encountering issues when trying to clone the Carla branch of Unreal Engine using the following command: `(git clone --depth 1 -b carla https://github.com/CarlaUnreal/UnrealEngine.git`).a branch for Unreal Engine.
How to fix it?

(Cloning into 'UnrealEngine'...
remote: Enumerating objects: 111593, done.
remote: Counting objects: 100% (111593/111593), done.
remote: Compressing objects: 100% (92142/92142), done.
error: RPC failed; curl 92 HTTP/2 stream 7 was not closed cleanly: CANCEL (err 8)
error: 4718 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output)

I also got the link from the ChatGpt, but it shows this error.
(C:\>git clone --depth 1 -b carla https://github.com/CarlaUnreal/UnrealEngine.git
Cloning into 'UnrealEngine'...
fatal: unable to access 'https://github.com/CarlaUnreal/UnrealEngine.git/': OpenSSL SSL_read: SSL_ERROR_SYSCALL, errno 0)

If I got the link from the ChatGPT then it shows different error.
(Cloning into 'UnrealEngine'...
fatal: unable to access 'https://github.com/CarlaUnreal/UnrealEngine.git/': OpenSSL SSL_read: SSL_ERROR_SYSCALL, errno 0)

PLease guide or help me to figure this out.

**Comment by OvalRooster** [comment]
I have the same

**Reply by Aimo2027** [reply]
Hi,

 I found this solution from the (Behnaz Fayazi) ."" In this case you need to check the capacity of your graphics card. For Carla you need to have access to minimum 6G GPU, more over about 20GB disk space, then you need to an IDE recommended Pycharm and make a virtual environment inside it and install Python version 3.7 and download Carla version like 0.9.13 (depending on Carla version try to install the suitable version of Python).
you can follow the rest of the instructions from my github: [GitHub - 3F999/CARLA: Simulation Autonomous Vehicles](https://github.com/3F999/CARLA)""

Thanks,
